src/libraries/System.Text.Encoding.CodePages/src/System/Text/EncodingNLS.cs

```diff
@@ -576,5 +576,50 @@ public override string WebName
                 50225 => "iso-2022-kr",
                 _ => WebName,
             };
+
+        public override int WindowsCodePage
+        {
+            get
+            {
+                (ushort FamilyCodePage, byte CodePageFlags) item = EncodingTable.GetCodePageItem(CodePage);
+                return item.FamilyCodePage;
+            }
+        }
+
+        public override bool IsBrowserDisplay
+        {
+            get
+            {
+                (ushort FamilyCodePage, byte CodePageFlags) item = EncodingTable.GetCodePageItem(CodePage);
+                return (item.CodePageFlags & EncodingTable.MIMECONTF_BROWSER) != 0;
+            }
+        }
+
+        public override bool IsBrowserSave
+        {
+            get
+            {
+                (ushort FamilyCodePage, byte CodePageFlags) item = EncodingTable.GetCodePageItem(CodePage);
+                return (item.CodePageFlags & EncodingTable.MIMECONTF_SAVABLE_BROWSER) != 0;
+            }
+        }
+
+        public override bool IsMailNewsDisplay
+        {
+            get
+            {
+                (ushort FamilyCodePage, byte CodePageFlags) item = EncodingTable.GetCodePageItem(CodePage);
+                return (item.CodePageFlags & EncodingTable.MIMECONTF_MAILNEWS) != 0;
+            }
+        }
+
+        public override bool IsMailNewsSave
+        {
+            get
+            {
+                (ushort FamilyCodePage, byte CodePageFlags) item = EncodingTable.GetCodePageItem(CodePage);
+                return (item.CodePageFlags & EncodingTable.MIMECONTF_SAVABLE_MAILNEWS) != 0;
+            }
+        }
```

Comment:
Can you please add triple slash comments on these new properties?

Reply:
This is internal class, do we still need to do that as the public properties already documented?

Reply:
Oh nevermind, I thought it was public.